src/types/swagger.ts

```diff
@@ -1175,6 +1178,8 @@ export interface components {
       }[];
       /** @description True if the account is an organization API service account */
       service_account?: boolean;
+      /** @description True if the account is an automated/manual test account */
```

Comment:
nit:
```suggestion
      /** @description True if the account is an test account */
```

The inclusion of "automated/manual" seems a little confusing/misleading here. Open to your thoughts on this though.

Reply:
I think github copilot wrote that lol

Reply:
Nope, that's taken from user.py line 43 actually, it just copied it I think

Reply:
You're right, let's keep it for now.